server/src/controllers/Order.ts

```diff
@@ -14,13 +14,13 @@ import { EntityFromBody } from 'typeorm-routing-controllers-extensions';
 
 import { Order } from '../entity/Order';
 
-const relations = ["menuItems"];
+const relations = ['menuItems'];
 
 @JsonController()
 export class OrderController {
   @Get('/orders/')
   public async getAll() {
-    return Order.find({ relations });
+    return Order.find({ relations: ['visit', 'menuItems'] });
```

Comment:
I would just make a `relations` variable at the top of the file which will make things easier to modify in the future